src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncTests.cs

```diff
@@ -13,11 +13,18 @@
 using Roslyn.Utilities;
 using Xunit;
 using Basic.Reference.Assemblies;
+using System.Runtime.CompilerServices;
+using System.Reflection;
+
+// PROTOTYPE: Verify execution of runtime async methods
+// PROTOTYPE: ILVerify for runtime async?
 
 namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
 {
     public class CodeGenAsyncTests : EmitMetadataTestBase
     {
+        private const MethodImplAttributes MethodImplOptionsAsync = (MethodImplAttributes)1024;
```

Comment:
Consider leaving PROTOTYPE marker to replace with real value once available here too